erna/hdf_utils.py

```diff
@@ -41,6 +41,25 @@ def rename_columns(columns):
     return [camel2snake(renames.get(col, col)) for col in columns]
 
 
+def add_theta_deg_columns(array):
```

Comment:
I would add a specific test to make sure this is an array and not an pandas dataframe, as this functinality seems to be needed for those as well. Actually it would be nice to have this for the other cases as well as we seem to be using them several times (at least here) maybe add a feature request to pyfact, and have the functions there?